client/my-sites/store/index.js

```diff
@@ -7,129 +7,9 @@ import { navigation, siteSelection, sites } from 'calypso/my-sites/controller';
 import isSiteStore from 'calypso/state/selectors/is-site-store';
 import { getSiteOption } from 'calypso/state/sites/selectors';
 import { getSelectedSite } from 'calypso/state/ui/selectors';
-import App from './app';
-import Dashboard from './app/dashboard';
 import StatsController from './app/store-stats/controller';
-import { bumpStat } from './lib/analytics';
-import StoreSidebar from './store-sidebar';
 import './style.scss';
 
-const getStorePages = () => {
-	const pages = [
-		{
-			path: '/store/:site',
-		},
-		{
-			path: '/store/products/:site',
-			statName: 'products',
-		},
-		{
-			path: '/store/product/:site',
-			statName: 'new-product',
-		},
-		{
-			path: '/store/product/:site/:product_id',
-			statName: 'edit-product',
-		},
-		{
-			path: '/store/products/categories/:site',
-			statName: 'product-categories',
-		},
-		{
-			path: '/store/products/category/:site/:category_id',
-			statName: 'edit-product-category',
-		},
-		{
-			path: '/store/products/category/:site',
-			statName: 'new-product-category',
-		},
-		{
-			path: '/store/orders/:site',
-			statName: 'orders',
-		},
-		{
-			path: '/store/orders/:filter/:site',
-			statName: 'orders',
-		},
-		{
-			path: '/store/order/:site/:order_id',
-			statName: 'order-details',
-		},
-		{
-			path: '/store/order/:site/',
-			statName: 'new-order',
-		},
-		{
-			path: '/store/promotions/:site',
-			statName: 'promotions',
-		},
-		{
-			path: '/store/promotion/:site',
-			statName: 'new-promotion',
-		},
-		{
-			path: '/store/promotion/:site/:promotion_id',
-			statName: 'edit-promotion',
-		},
-		{
-			path: '/store/reviews/:site',
-			statName: 'reviews',
-		},
-		{
-			path: '/store/reviews/:filter/:site',
-			statName: 'reviews',
-		},
-		{
-			path: '/store/reviews/:product_id/:filter/:site',
-			statName: 'reviews',
-		},
-		{
-			path: '/store/settings/:site',
-			statName: 'payment-settings',
-		},
-		{
-			path: '/store/settings/payments/:site',
-			statName: 'payment-settings',
-		},
-		{
-			path: '/store/settings/shipping/:site',
-			statName: 'shipping-settings',
-		},
-		{
-			path: '/store/settings/shipping/zone/:site/:zone?',
-			statName: 'shipping-settings',
-		},
-		{
-			path: '/store/settings/taxes/:site',
-			statName: 'tax-settings',
-		},
-		{
-			path: '/store/settings/email/:site/:setup?',
-			statName: 'email',
-		},
-	];
-
-	return pages;
-};
-
-function getAnalyticsPath( path, params ) {
-	if ( '/store/settings/:site' === path ) {
-		return '/store/settings/payments/:site';
-	}
-
-	if ( '/store/settings/email/:site/:setup?' === path ) {
-		return params.setup ? '/store/settings/email/:site/:setup' : '/store/settings/email/:site';
-	}
-
-	if ( '/store/settings/shipping/zone/:site/:zone?' === path ) {
-		return params.zone
-			? '/store/settings/shipping/zone/:site/:zone'
-			: '/store/settings/shipping/zone/:site';
-	}
-
-	return path.replace( '?', '' ).replace( ':filter', params.filter );
-}
-
 function notFoundError( context, next ) {
```

Comment:
You can also remove this `notFoundError` component and the associated `content-404` styles. Calypso has a default handler for 404 pages, and this one doesn't add anything new. It would be another matter if it provided some smart links to WC Admin etc, but it doesn't.